sci-physics/fastjet-contrib/fastjet-contrib-1.049.ebuild

```
@@ -0,0 +1,43 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MY_PN=fjcontrib
+MY_P=${MY_PN}-${PV}
+
+DESCRIPTION="3rd party extensions of FastJet."
+HOMEPAGE="http://fastjet.hepforge.org/contrib/"
+SRC_URI="http://fastjet.hepforge.org/contrib/downloads/${MY_P}.tar.gz"
+S=${WORKDIR}/${MY_P}
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
+
+DEPEND=">=sci-physics/fastjet-3.4.0"
+RDEPEND="${DEPEND}"
+
+PATCHES=(
+"${FILESDIR}"/${PN}-soname.patch
+)
+
+src_prepare() {
+	default
+}
+
+src_configure() {
+	./configure --prefix=/usr --fastjet-config=/usr/bin/fastjet-config CFLAGS="${CFLAGS}" CXXFLAGS="${CXXFLAGS}" FFLAGS="${FFLAGS}" LDFLAGS="${LDFLAGS}"
+}
+src_compile() {
+	emake
+	emake fragile-shared
+}
+
+src_install() {
+	emake install PREFIX="${D}/usr"
```

```suggestion
	emake install PREFIX="${ED}/usr"
```
ED also takes the EPREFIX into account